events/javaone/2015/_common/javaone.yml

```diff
@@ -1,61 +1,61 @@
 minibooth_sessions:
   - title: 'Standardized Extension-Building in Java EE with CDI and JCA'
-    url: http://developers.redhat.com/video/youtube/I-Po2phF-_M
+    url: /video/youtube/I-Po2phF-_M
```

Comment:
This isn't right. See: http://it-developers-pr.stage.redhat.com/pr/673/build/387/events/javaone/2015/. Notice links to videos on http://it-developers-pr.stage.redhat.com/video/youtube/I-Po2phF-_M that should be staying on this PR.